CMakeLists.txt

```diff
@@ -481,7 +534,9 @@ if (VITA_BUILD)
     )
 else()
     target_link_libraries (${SHORT_NAME} ${SDL2_LIBRARY} ${SDL2_MIXER_LIBRARY})
-    install(TARGETS ${SHORT_NAME} RUNTIME DESTINATION bin)
+    if(NOT APPLE)
+        install(TARGETS ${SHORT_NAME} RUNTIME DESTINATION bin)
+    endif()
```

Comment:
On macOS, install() is already called once before, so we do not need to call it again. (calling it twice actually causes an error on a call to "install_name_tool")